### DISCUSSION TITLE
无法init did aggregator

### DISCUSSION THREAD

**Discussion by xiegudong45** [NonceGeek/Move-Camp#211] [Q&A] [answered]
问题：点击init addraggr in did contract后显示transaction error
报错如下：
![image](https://user-images.githubusercontent.com/30063330/202838729-657e7fa8-4469-427b-9895-0faf51fe944d.png)
![image](https://user-images.githubusercontent.com/30063330/202838735-aa56eda4-853c-471c-9375-be250e0774da.png)
地址设置检查过没有问题
![image](https://user-images.githubusercontent.com/30063330/202838784-64a302d3-e0fd-4b8d-b870-52a023d469f1.png)
![image](https://user-images.githubusercontent.com/30063330/202838796-a69ae010-2eb9-4a16-8151-b1d24dcfe130.png)

**Comment by leeduckgo** [comment, marked as answer]
目前版本可用，但待更新
